rep/processors/CtGovPdfProcessor.py

```diff
@@ -16,13 +17,23 @@ def __init__(self):
         self.VOTE_FOR_PATTERN = r"(Vote for )(.*?)( Seq)"
         self.UNKOWN_PATTERN = r"[a-zA-Z .]"
 
-    def process_blob(self, blob):
+    def process_blob(self, blob, sourceUrl):
         try: 
-            self._process_blob_wrapper(blob)
-        except Exception:
-            raise PdfProcessorException("Unable to process pdf")
+            self._process_blob_wrapper(blob, sourceUrl)
+        except IndexError:
+            raise PdfProcessorException("Unable to process -- PDF does not contain vote data")
+
+    def mark_processed(self, sourceUrl):
```

Comment:
so even though python classes don't really have private members, it's standard practice to prefix methods that you don't want to be part of the classes public interface with an underscore. So if we want processors to have a `mark_processed` method, then you'd want to add this to the `BaseProcessor` interface. This looks like it should be a private method though, so I'd just change it to `_mark_processed`

Reply:
Ok sure, I wasn't clear on when to prefix with an underscore, but I'll change that